bin/configure-tower-projects.py

```diff
@@ -52,6 +52,15 @@
     "m6i.8xlarge", "r5a.8xlarge", "r6a.8xlarge", "r6i.8xlarge"
 )
 
+GPU_EC2_INSTANCE_TYPES = (
```

Comment:
The CI cannot update compute environments, it can only create them therefore we need to delete the compute envs to allow the CI to create new ones with these new instance types.  Instead of updating all compute environments in every nextflow project we are going spot update by only removing the ones that need updating.  Which nextflow projects will need these new instance types @vpchung ?

Reply:
Thanks, @zaro0508 - there are currently no nextflow projects that are currently utilizing GPUs. So we're good 👍🏼